internal/drivers/basic/account_management.go

```diff
@@ -0,0 +1,137 @@
+/******************************************************************************
+*
+*  Copyright 2024 SAP SE
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+******************************************************************************/
+
+package basic
+
+import (
+	"os"
+	"sync"
+
+	"github.com/sapcc/go-bits/osext"
+
+	"github.com/sapcc/keppel/internal/keppel"
+	"github.com/sapcc/keppel/internal/models"
+
+	"gopkg.in/yaml.v2"
+)
+
+// AccountManagementDriver is the account management driver "basic".
+type AccountManagementDriver struct {
+	ConfigPath string
+	config     AccountConfig
+	lock       sync.RWMutex
+}
+
+type AccountConfig struct {
+	Accounts []Accounts `yaml:"accounts"`
+}
+
+type Accounts struct {
+	Name                 string                      `yaml:"name"`
+	AuthTenantID         string                      `yaml:"auth_tenant_id"`
+	GCPolicies           []keppel.GCPolicy           `yaml:"gc_policies"`
+	RBACPolicies         []keppel.RBACPolicy         `yaml:"rbac_policies"`
+	ReplicationPolicy    keppel.ReplicationPolicy    `yaml:"replication_policy"`
+	SecurityScanPolicies []keppel.SecurityScanPolicy `yaml:"security_scan_policies"`
+	ValidationPolicy     keppel.ValidationPolicy     `yaml:"validation_policy"`
+}
+
+func init() {
+	keppel.AccountManagementDriverRegistry.Add(func() keppel.AccountManagementDriver {
+		return &AccountManagementDriver{}
+	})
+}
+
+// PluginTypeID implements the keppel.AccountManagementDriver interface.
+func (a *AccountManagementDriver) PluginTypeID() string { return "basic" }
+
+// Init implements the keppel.AccountManagementDriver interface.
+func (a *AccountManagementDriver) Init() error {
+	configPath, err := osext.NeedGetenv("KEPPEL_ACCOUNT_MANAGEMENT_FILE")
```

Comment:
The common naming convention would be:
```suggestion
	configPath, err := osext.NeedGetenv("KEPPEL_ACCOUNT_MANAGEMENT_CONFIG_PATH")
```